### PR TITLE
#34 #35

### PR DESCRIPTION
**Pull request by VIad0s**
Create forum_project and define apps

### CODE REVIEW

**Comment by 0leksandrr** [comment]
You need to move all the credentials to environment variables and create a .env_example file.

Example .env 

DJANGO_ENV=DEVELOPMENT
DJANGO_LOG_LEVEL=INFO
DJANGO_SECRET_KEY=
DJANGO_ALLOWED_HOSTS=
DJANGO_CORS_ALLOWED_ORIGINS=

# ===== DATABASE =====
DB_NAME=
DB_USER=
DB_PASSWORD=
DB_HOST=db
DB_PORT=5432

# ===== LOCAL POSTGRES DOCKER =====
POSTGRES_DB=postgres
POSTGRES_USER=postgres
POSTGRES_PASSWORD=postgres

File: Forum/settings.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = os.getenv('DJANGO_ENV') == 'DEVELOPMENT'
+
+ALLOWED_HOSTS = [os.getenv('DJANGO_ALLOWED_HOSTS').split(',')]
```
Comment (mehalyna) [inline Forum/settings.py:29]: `ALLOWED_HOSTS = os.getenv('DJANGO_ALLOWED_HOSTS', '').split(',')`

File: Forum/settings.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+CORS_ALLOWED_ORIGINS = os.getenv('DJANGO_CORS_ALLOWED_ORIGINS').split(',')
```
Comment (mehalyna) [inline Forum/settings.py:30]: `CORS_ALLOWED_ORIGINS = os.getenv('DJANGO_CORS_ALLOWED_ORIGINS', '').split(',')`

File: Forum/settings.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = os.getenv('DJANGO_ENV') == 'DEVELOPMENT'
```
Comment (mehalyna) [inline Forum/settings.py:27]: `DEBUG = os.getenv('DJANGO_ENV', 'PRODUCTION') == 'DEVELOPMENT'`

**Review by mehalyna — CHANGES_REQUESTED** [review]

**Pull request merged**